### PR TITLE
DOCSP-25130-sharded-data-method

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
## DESCRIPTION

Adds mongosh method sh.getShardedDataDistribution.

## STAGING

https://preview-mongodbjasonpricemongodb.gatsbyjs.io/mongodb-shell/DOCSP-25130-sharded-data-method/reference/methods/#sharding-methods

## JIRA

https://jira.mongodb.org/browse/DOCSP-25130

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=659600d6dcc0144a51e0310e

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/methods.txt [hunk 1]
```diff
@@ -1334,6 +1334,13 @@ Sharding Methods
      - Returns a boolean to report if the :term:`balancer` is currently
        enabled.
 
+   * - :method:`sh.getShardedDataDistribution()`
+
```
Comment (ianf-mongodb) [inline source/reference/methods.txt:1338]: **Suggestion** 

You might want to add a comment here that the function definition is documented in both the docs-mongodb-shell and docs-mongodb-internal repos for future updates.

**Review by ianf-mongodb — APPROVED** [review]
LGTM Cheers @jason-price-mongodb

**Comment by jason-price-mongodb** [comment]
@addaleax please review. Thanks.

**Review by ianf-mongodb — APPROVED** [review]
LGTM

**Review by addaleax — APPROVED** [review]

**Pull request merged**